src/Tickets/Emails/Hooks.php

```diff
@@ -69,6 +70,10 @@ protected function add_filters() {
 		add_filter( 'tribe_tickets_admin_manager_request', [ $this, 'filter_add_preview_modal_content' ], 15, 2 );
 
 		add_filter( 'wp_redirect', [ $this, 'filter_redirect_url' ] );
+
+		// Legacy emails.
```

Comment:
This is not adding any clarity at all. It's not just legacy that we're hooking on to. It's the point where emails are being delivered.

The point of having legacy methods in the modules class was to quickly deprecate the old emails whenever we wanted by deprecating that new `_legacy` function that was just holding how the legacy emails were sent.

Reply:
Modified the comment for a bit more clarity.